src/components/ReleaseMonitor/ReleaseListHeader.tsx

```diff
@@ -0,0 +1,35 @@
+import { createTableHeaders } from '../../shared/components/table/utils';
+
+export const releaseTableColumnClasses = {
+  name: 'pf-m-width-20 ',
+  status: 'pf-m-width-20',
+  completionTime: 'pf-m-width-20',
+  component: 'pf-m-width-20',
+  application: 'pf-m-width-20',
+  releasePlan: 'pf-m-width-20',
+  namespace: 'pf-m-width-20 ',
```

Comment:
ideally, the total width should be 100%, right now it's 140 - it may cause layout issues in certain screen sizes